mw3071.md

```diff
@@ -0,0 +1,18 @@
+<h1>Open Source Development</h1>
+  
+Name: Matthew Wang
+
+UNI: mw3071
+
+GitHub: [link](https://github.com/mw3071)
+
+GitHub README: [link](https://github.com/mw3071/mw3071/blob/main/README.md)
+
+<h2>Project Proposal</h2>
+
+[Columbia Discord Bot](https://github.com/mw3071/project-proposals-s2023/blob/main/projects/python/columbia-discord-bot.md)
```

Comment:
use a relative link to this repository, like `./projects/python/columbia-discord-bot.md`